docs(fix): Improve language and clarify that `cssFloat` is only needed in older browsers (#32069)
mdn/content: files/en-us/web/api/htmlelement/style/index.md

> **Note:** CSS property names are converted to JavaScript identifier with these rules:
>

> > - If the property is made of one word, it remains as it is: `height` stays as is (in lowercase). `float`, however, is a reserved keyword in JavaScript. While modern browsers support the direct use of `float` in JavaScript to access the `float` CSS property, this property name was historically converted to `cssFloat`. `cssFloat` is used in older browsers and still supported as an alias in modern browsers.

Reviewer: The "however" felt off in the second sentence, so reordered it again.  I am not sure if the `is` in the last sentence would be better as `was`.

```suggestion
> - If the property is made of one word, it remains as it is: `height` stays as is (in lowercase). As `float` is a reserved keyword in JavaScript, this property name was historically converted to `cssFloat`. Modern browsers all now support the direct use of `float` in JavaScript to access the `float` CSS property, but `cssFloat` is used in older browsers and is still supported as an alias in modern browsers.
```

Author: This looks good, thank you!! Moved "all" to the beginning but accepting rest of your wording suggestion. "is" in the last sentence seems good.

>
> - If the property is made of one word, it remains as it is: `height` stays as is (in lowercase).
> - If the property is made of several words, separated by dashes, the dashes are removed and it is converted to {{Glossary("camel_case", "camel case")}}: `background-attachment` becomes `backgroundAttachment`.

> > - The property `float`, being a reserved JavaScript keyword, is converted to `cssFloat`. This is true in older browsers. Modern browsers support the direct use of `float` in JavaScript to access the `float` CSS property.

Reviewer: If it is not an issue anymore, we should start off with the current status then provide the outdated stuff, not the other way around. Is this better?
```suggestion
> - `float` is a reserved JavaScript keyword. While modern browsers support the direct use of `float` in JavaScript to access the `float` CSS property, this property name was originally converted to `cssFloat`. This is used in older browsers and still supported as an alias in modern browsers.
```

Author: Makes sense.

Also, since with `float` we're still referring to the first rule in the list, I have added it to the first list item (removed the last item specific to `float`). What do you think about (originally -> historically?):

> - If the property is made of one word, it remains as it is: `height` stays as is (in lowercase). `float`, however, is a reserved keyword in JavaScript. While modern browsers support the direct use of `float` in JavaScript to access the `float` CSS property, this property name was historically converted to `cssFloat`. `cssFloat` is used in older browsers and still supported as an alias in modern browsers.
